JS/models/ExperimentalTypes.js

```diff
@@ -1,8 +1,8 @@
 const ExperimentalTypes   = {
-    LIFTING_BODY: 'lifting_body',
+    LIFTING_BODY: 'LIFTING_BODY',
     HYPERSONIC: 'HYPERSONIC',
     HIGH_ALTITUDE: 'HIGH_ALTITUDE',
-    VTOL: "VTOL"
+    VTOL: "VTOL" // A vertical take-off and landing (VTOL) aircraft
```

Comment:
Please do not use descriptive comments. The variable name should be descriptive by itself.

Reply:
Done. Variable was renamed throughout the project scripts